packages/babel-plugin-transform-spread/src/index.ts

```diff
@@ -33,15 +44,41 @@ export default declare((api, options) => {
     return false;
   }
 
-  function push(_props, nodes) {
+  function push(
+    _props: Array<
+      | t.Expression
+      | t.SpreadElement
+      | t.JSXNamespacedName
```

Comment:
Q: When can this be a `JSXNamespacedName`?

Reply:
`CallExpression.arguments` and `NewExpression.arguments` type is `(t.Expression | t.SpreadElement | t.JSXNamespacedName | t.ArgumentPlaceholder)[]`, but this cannot be `t.JSXNamespacedName` nor `t.ArgumentPlaceholder` so I fixed. 

https://github.com/babel/babel/pull/13770/commits/fdab8e01226f52e6e99e36258015e59a2eb1cb38
>const args = node.arguments as Array<t.Expression | t.SpreadElement>;